tools/sof-dump-status.py

```diff
@@ -331,6 +382,14 @@ def _getStr(tmp_dict, key):
         print(sysinfo.sys_power['run_status'][ret_args['dsp_status']]['status'])
         exit(0)
 
+    if ret_args.get('dapm') is True:
+        if os.environ['USER'] != 'root': # this operation need root permission
```

Comment:
This Code is duplicated. Do we need this double check?

Reply:
original define `loadDAPM` can load by default,  
but I find the content is too many for dump, so I just add it into this option  

So let's keep it, when decide to filter the target keyword for the content, like just keep **on**